### ISSUE TITLE
pre-trained model

### ISSUE THREAD

**Issue by leatherking** [GMvandeVen/brain-inspired-replay#9]
Hi, I want to ask about the pre-trained models. The paper says you pre-train on cifar10, but the default pre-trained model, in the store/models/ file, C3-5×16-bn, seems a model of pre-training on cifar100. I have run main_pretrain.py on cifar10 to change the pre-trained model and it worked worse than the default. Did you use pre-trained model of cifar100 rather than cifar10 on paper's experiments?

**Comment by GMvandeVen** [comment]
Hi, thanks for your question. The short answer is that the provided pre-trained convolutional model `store/models/C3-5x16-bn` was the pre-trained model that was used in the paper, and this model was pre-trained on CIFAR-10 (not CIFAR-100).

That said, I should note a few things.
Firstly, for the pre-training on CIFAR-10, data augmentation was used, but I realize this is not properly described in the method section of the paper itself. Apologies for that. The command used for the pre-training is provided here: https://github.com/GMvandeVen/brain-inspired-replay/blob/ce30785bd1451b768bacb17c819a4236cd06389a/create_figures.sh#L30 (Note that for the incremental training on CIFAR-100 data augmentation was not used.)
Secondly, I now realize that there can be quite a variability in how suitable the resulting pre-trained convolutional layers from the above command are for incremental training on CIFAR-100. In the paper, I ran each method for multiple random seeds, but I always used the same pre-trained convolutional layers. In hindsight, it might have been better to also use different pre-trained convolutional layers for each random seed.

**Comment by leatherking** [comment]
Many thanks, I get it.
